### PR TITLE
[risk=low][RW-8985] remove deprecated methods

### PR DESCRIPTION
**Pull request by jmthibault79**
After #7090 and a release, UserAdminController.bypassAccessRequirement() and  UserService.updateBypassTime() can be removed
- also remove (unrelated) deprecated anyModuleHasExpired

Tested locally by executing all the affected calls which aren't deleted here.

Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/UserAdminController.java [hunk 1]
```diff
@@ -28,30 +30,34 @@
...
+    this.taskQueueService = taskQueueService;
     this.userProvider = userProvider;
+    this.userService = userService;
     this.workbenchConfigProvider = workbenchConfigProvider;
-    this.taskQueueService = taskQueueService;
   }
 
   @Override
   @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
+  @Deprecated // use updateAccountProperties()
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/api/UserAdminController.java:60]: updateAccountProperties() does everything this endpoint does (plus other things).

It also allows bypassing multiple modules with a single call. bypassAccessRequirement() only changes a single module.

File: api/src/main/java/org/pmiops/workbench/api/UserAdminController.java [hunk 2]
```diff
@@ -98,8 +104,9 @@ public ResponseEntity<EmptyResponse> unsafeSelfBypassAccessRequirement(
     if (!workbenchConfigProvider.get().access.unsafeAllowSelfBypass) {
       throw new ForbiddenException("Self bypass is disallowed in this environment.");
     }
-    long userId = userProvider.get().getUserId();
-    userService.updateBypassTime(userId, request);
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/api/UserAdminController.java:102]: this is now a pass-through to accessModuleService.updateBypassTime()

File: api/src/main/java/org/pmiops/workbench/profile/ProfileService.java [hunk 3]
```diff
@@ -134,19 +133,8 @@ public Profile getProfile(DbUser userLite) {
         accessTierService.getAccessTierShortNamesForUser(user);
     final List<UserTierEligibility> userTierEligibilities =
         institutionService.getUserTierEligibilities(user);
-
-    final List<AccessModuleStatus> accessModuleStatuses =
-        accessModuleService.getAccessModuleStatus(userLite);
-    // TODO RW-8236 Remove anyModuleHasExpired after release.
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/profile/ProfileService.java:140]: The release in question was in **May** so I think we're safe to remove this now

File: api/src/main/java/org/pmiops/workbench/api/UserAdminController.java [hunk 4]
```diff
@@ -98,8 +95,9 @@ public ResponseEntity<EmptyResponse> unsafeSelfBypassAccessRequirement(
...
+    final DbUser user = userProvider.get();
+    accessModuleService.updateBypassTime(user.getUserId(), request);
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/api/UserAdminController.java:99]: TODO soon: call `updateUserAccessTiers()` from `updateBypassTime()` and `updateCompletionTime()`

**Review by NehaBroad — APPROVED** [review]

**Pull request merged**